recipes/sos-r/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "sos-r" %}
+{% set version = "0.19.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ed48f4f32386a9b6a7e2c13056ce43d6ff151dc33afea5a6b96316b3af0df43b
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - ipython
+    - notebook >=5.0.0
```

Comment:
Are ipython and notebook required here? They are not part of `install_requires` in setup.py.